doc/command-line.md

```diff
@@ -92,11 +94,34 @@ Finally, there are a number of utility and information commands:
 Please use the command line option `--help` to get more detailed usage
 information for each command.
 
-Note that as of sourmash v3.4, most commands will load signatures from
+Note that as of sourmash v3.4, all commands should load signatures from
 indexed databases (the SBT and LCA formats) as well as from signature files.
 
+### `sourmash sketch` - make sourmash signatures from sequence data
+
+Most of the commands in sourmash work with **signatures**, which contain information about genomic or proteomic sequences. Each signature contains one or more **sketches**, which are compressed versions of these sequences. Using sourmash, you can search, compare, and analyze these sequences in various ways.
```

Comment:
Should the detail from compute, below, be moved to this section instead?

```
It takes as input FASTA or FASTQ files, and these files can be uncompressed or compressed with gzip or bzip2. 
```

Reply:
great suggestion, done!